web/source/keyboards/kmwkeyboards.ts

```diff
@@ -477,7 +477,12 @@ namespace com.keyman.keyboards {
       // if(!PLgCode && (<KeymanBase>keymanweb).keyboardManager.activeStub) {
       //   PLgCode = (<KeymanBase>keymanweb).keyboardManager.activeStub['KLC'];
       // }
-      this.doKeyboardChange(PInternalName, PLgCode);
+      let _this = this;
```

Comment:
```suggestion
      const _this = this;
```

Let's go with using `const` where possible :grin: -- it transpiles down to `var` anyway...